HISTORY.md

```diff
@@ -1,5 +1,35 @@
 # Release Notes
 
+## 1.10.0 - 2024-02-15
+
+This release adds multiple improvements to handling premium transformers and column relationships, including using the GPSNoiser as the default transformer for GPS column relationships and using premium transformers even if the PII flag is set to true. Additionally, the SDV now warns users to save the metadata after auto-detection has been used. Semantic sdtype detection has also been improved to tokenize column names to prevent unexpected substring matches.
```

Comment:
maybe we leave the GPSNoiser part out since it will only work if you have enterprise